README.md

```diff
@@ -392,7 +392,7 @@ instance = SingleVMOpenEdXInstance.objects.create(
     sub_domain='dogwood.sandbox',
     name='Dogwood',
     fork_name='edx/edx-platform',
-    branch_name='named-release/dogwood',
+    branch_name='named-release/dogwood.rc',
```

Comment:
The git command-line interface generally accepts a "rev" or revision specifier everywhere.  "Refs" or references are just one kind of "rev" (which is slightly confusing).  The particular endpoint of the Github API that we are using doesn't give that flexibility, but there is a [different endpoint](https://developer.github.com/v3/repos/commits/#get-a-single-commit) that does:

https://api.github.com/repos/edx/edx-platform/commits/named-release/dogwood.3
https://api.github.com/repos/edx/edx-platform/commits/named-release/dogwood.rc

If we use this endpoint, we won't need to specify the `ref_type` anymore, and any git "rev" will just work (i.e. everything documented in the [gitrevisions man page](https://www.kernel.org/pub/software/scm/git/docs/gitrevisions.html)), so I suggest doing that to avoid the confusion.  +1 on waiting until #63 is merged before any further changes.